crates/revm/src/db/states/bundle_state.rs

```diff
@@ -581,6 +581,68 @@ impl BundleState {
         self.reverts.push(reverts);
     }
 
+    /// Generate a [`StateChangeset`] from the bundle state without consuming
+    /// it.
+    pub fn to_plain_state(&self, is_value_known: OriginalValuesKnown) -> StateChangeset {
```

Comment:
looks like there is no benefit of self ownership at all, so i would just remove the into method entirely, or deprecate + delegate to avoid breaking changes